config_human.tsv

```diff
@@ -0,0 +1,17 @@
+local_dir	"/home/alexandra/Documents/Data/Generic_expression_patterns/"
```

Comment:
I am using this in both R and python functions.

Interesting idea...perhaps we can chat offline about this. I'm not sure if I'm exactly understanding the benefit compared to using it in this config file...so i'm sure i'm missing something here

Reply:
Ok, I think if you're planning to use this in your R scripts too, what you have is fine.

In your config file you repeat `/home/alexandra/Documents/Data/Generic_expression_patterns/` in lots of places - this kind of repetition just makes me nervous in general, since you have to remember to change it in all of those places if you move the repo to a new directory. Just trying to think of ways to avoid that and have a single point of truth for where the repo is located, but I don't know of a good way to do that in a text file.

Happy to chat offline if this is still unclear.

Reply:
Ah..I see now. I think this config file was a nice step up from previously me defining this dir in each separate notebook. So at least the change is happening within this one config file. But I agree that it would be even better if there is a way to have no repetition within that file too. For now I think I'll leave it, but keep it in mind moving forward for sure